packages/headless/src/test/mock-recommendation-v2.ts

```diff
@@ -0,0 +1,20 @@
+import {RecommendationCommerceSuccessResponse} from '../api/commerce/common/response';
+import {QueryRecommendationCommerceAPIThunkReturn} from '../features/commerce/common/actions';
+
+export function buildFetchRecommendationV2Response(
```

Comment:
`Mock` would be a better name to standardize with the other functions in the `/test` folder.

```suggestion
export function buildMockRecommendationV2Response(
```